### PR TITLE
docs: Use the Node.js example app as Docker push example

### PR DESCRIPTION
**Pull request by lmars**
The Node.js app conforms to the expectations of a Flynn app (i.e. it listens on the PORT environment variable) so provides a better example than ElasticSearch.

I have also clarified the routing setup.

Closes #3222.

### CODE REVIEW

File: docs/content/docker.md [hunk 1]
```diff
@@ -34,29 +34,42 @@ above the error then re-run `flynn docker login`._
...
-Here is an example of deploying the `elasticsearch:2.3.3` image from
-[Docker Hub](https://hub.docker.com/_/elasticsearch/).
+Flynn automatically registers the HTTP route `http://APPNAME.$CLUSTER_DOMAIN`
+for the app. In order to receive HTTP traffic for this route, the app needs to
+listen on the port which is set in the `PORT` environment variable.
 
-Pull the image from Docker Hub:
+## Example
+
+Here is an example of deploying the [Flynn NodeJS example app](https://github.com/flynn-examples/nodejs-flynn-example)
```
Comment (titanous) [inline docs/content/docker.md]: s/NodeJS/Node.js

File: docs/content/docker.md [hunk 2]
```diff
@@ -89,32 +103,29 @@ config.
...
-  "version" : {
-    "number" : "2.3.3",
-    "build_hash" : "218bdf10790eef486ff2c41a3df5cfa32dadcfde",
-    "build_timestamp" : "2016-05-17T15:40:04Z",
-    "build_snapshot" : false,
-    "lucene_version" : "5.5.0"
-  },
-  "tagline" : "You Know, for Search"
-}
+$ curl http://nodejs.$CLUSTER_DOMAIN
```
Comment (titanous) [inline docs/content/docker.md]: s/$CLUSTER_DOMAIN/demo.localflynn.com

**Comment by lmars** [comment]
@titanous comments addressed

**Comment by titanous** [comment]
LGTM

**Pull request merged**